compiler/src/iree/compiler/API/generate_exports.py

```diff
@@ -61,6 +61,7 @@
     "Transforms.h",
     "Dialect/GPU.h",
     "Dialect/Linalg.h",
+    "Dialect/LLVM.h",
```

Comment:
We should run this script after changes to it. Looks like CI (and my local Windows build) passed, but we've had issues before where other builds fail afterwards. Safest to just run the script as part of this PR.

```
python compiler/src/iree/compiler/API/generate_exports.py
git add -A
git commit ...
```

Reply:
Sounds good. I have updated the PR with the files modified by that script.